tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,183 @@
+import axios from 'axios';
+import * as fs from 'fs';
+import * as https from 'https';
+import jwtDecode from 'jwt-decode';
+
+import test, { expect } from '../fixtures/fixtures';
+
+test.describe('SAML Login', () => {
+  test.use({ storageState: 'storage.json' });
+  test.describe('SAML login via sso/id/login', () => {
+    test('Logs in using UI', async ({ environment, context, baseUrl, page, password, username }) => {
+      test.skip(environment !== 'staging');
+      test.setTimeout(240000);
+      const saml_settings={
+        'login':'morty',
+        'password':'panic',
+        'email':'msmith@samltest.id',
+        'idp_url':'https://samltest.id/saml/idp'
+      };
+      const timeout_4s=4000;
+      const timeout_1s=1000;
+      const storage=await context.storageState();
+      const jwt=storage['cookies'].find(cookie => cookie.name === 'JWT').value;
+
+      const idp_response = await axios({
+        url: saml_settings['idp_url'],
+        method: 'GET'
+      });
+
+      if (idp_response.status !== 200) {
+        throw 'expected 201';
+      }
```

Comment:
```suggestion
  expect(idp_responose.status).toBe(200);
```
or `expect(idp_responose.status).toBe(201);` - not quite sure here 😉

Reply:
fixed everywhere. heh, yeah, throw what you dont really expect ;-)